README.md

```diff
@@ -378,3 +378,11 @@ Credit: @axelf4
 ```vim
 :echom test_null_list()
 ```
+
+## I will do it for you (Free service)
```

Comment:
I think the section title is wrong, it should be: _Quit as a service_.

Reply:
QaaS